recipes/co/all/conanfile.py

```diff
@@ -30,75 +32,76 @@ class CoConan(ConanFile):
         "with_libcurl": False,
         "with_openssl": False,
     }
+    deprecated = "cocoyaxi"
 
-    _cmake = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    def export_sources(self):
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
 
     def requirements(self):
         if self.options.with_libcurl:
-            self.requires("libcurl/7.79.1")
+            self.requires("libcurl/[>=7]")
```

Comment:
Not allowed, please pin it for now :)

BUT, I'm discussing with the team on allowing it for libcurl as it has a solid compatibility behaviour between versions, will let you know once we come to a conclussion!